source/stylesheets/screen.css.scss

```
@@ -1,2 +1,9 @@
 @import "govuk_tech_docs";
 @import "modules/page-banner";
+
+a[href^="https://sites.google.com/a/digital.cabinet-office.gov.uk/"]::after,
+a[href^="https://sites.google.com/cabinetoffice.gov.uk/"]::after,
+a[href^="https://gds.slack.com/"]::after
+{
+  content: " (GDS staff only)";
```

As discussed on Slack: this CSS appends a non-breaking space and a [Unicode padlock](https://unicodeplus.com/U+1F512) to the links in scope of the rule.

```suggestion
  content: " "\00a0\01F512";
```

Tested in this shonky bit of HTML:

```
<html>
<head>
<style type="text/css">
a::after {
    content: "\00a0\01F512";
}
</style>
</head>
<body>
<a href="">Link</a>
</body>
</html>
```

---

😅 ecb7c30e4f28ab1802c4475d0ef89e8ba04d5703